Docs/articles/Kernel/Audio.md

```diff
@@ -0,0 +1,81 @@
+# The Cosmos Audio Infrastructure (CAI)
```

Comment:
Great article! It makes it very clear of how to use CAI.
Can you add it to https://github.com/CosmosOS/Cosmos/blob/master/Docs/articles/toc.md so that it can be found?